opengeode/opengeode.py

```diff
@@ -614,10 +614,10 @@ def recursive_render(content, dest_scene):
                     try:
                         if s.ast.errors or s.ast.warnings:
                             # Keep track of all symbols containing errors,
-                            # they are processesd in "find_symbols_and_update_errors"
+                            # they are processed in "find_symbols_and_update_errors"
                             G_ERRORS.append(s)
                     except Exception as e:
-                        print("Ooops", str(e))
+                        print("Oops", str(e))
```

Comment:
This repository doesn't appear to be very "cute", so there didn't seem to be a big reason not to make this change.